react/components/QuoteDetails.tsx

```diff
@@ -573,13 +619,35 @@ const QuoteDetails: FunctionComponent = () => {
                           },
                           minWidth: 300,
                         },
+                        listPrice: {
+                          title: formatMessage(messages.originalPrice),
+                          headerRight: true,
+                          width: 120,
+                          cellRenderer: ({ rowData }: any) => {
+                            return (
+                              isSalesRep && (
+                                <div
+                                  className={`w-100 tr${
+                                    rowData.listPrice !== rowData.sellingPrice
+                                      ? ' strike '
+                                      : null
```

Comment:
This is resulting in "null" being written into the class attribute.
```suggestion
                                      : ''
```